iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -35,8 +35,21 @@ public enum ErrorCode
         [EditorBrowsable(EditorBrowsableState.Never)]
         DeviceInvalidResultCount = 400002,
 
+        /// <summary>
+        /// The client has requested an operation that the hub recognizes as invalid. Check the error message
+        /// for more information about what is invalid.
+        /// </summary>
+        // Note: although infrequent, this does appear in logs for "Amqp Message.Properties.To must contain the device identifier".
+        // and perhaps other cases.
         InvalidOperation = 400003,
+
+        /// <summary>
+        /// Something in the request payload is invalid. Check the error message for more information about what
+        /// is invalid.
+        /// </summary>
+        // Note: one example found in logs is for invalid characters in a twin property name.
```

Comment:
Should we mention that we found it in logs? It is probably not relevant to the customer where we found it.

Reply:
Yeah, not sure how to word this. It is a code comment for us. I don't specify what logs, just logs, so I thought maybe being ambiguous was sufficient.

Thoughts?

Reply:
I missed that it was just a code comment and not in the summary. Seems reasonable to keep it as a code comment for our reference.